crates/core/tests/time_travel.rs

```diff
@@ -13,6 +15,11 @@ async fn time_travel_by_ds() {
         ("00000000000000000002.json", "2020-05-03T22:47:31-07:00"),
         ("00000000000000000003.json", "2020-05-04T22:47:31-07:00"),
         ("00000000000000000004.json", "2020-05-05T22:47:31-07:00"),
+        // Final file is uncommitted by Spark and is in a .tmp subdir
```

Comment:
Since this fix ignores _any_ sub-directory, the test should also contain a testcase of a sub-directory not starting with a dot.

Reply:
I don't understand why we need a second test case? The name of the subdirectory does not matter, any and all files in subdirectories are ignored. I only used the name `.tmp` to clarify what is going on / better match what spark does.